test/testSynergy.jl

```diff
@@ -4,14 +4,14 @@
         FcExpr = ones(5) * 1e3
 
         # All the receptors are the same, so this should be flat
-        output = FcgR.calculateIsobologram(1, 5, 16, 1.2e-9, FcExpr, Kav, quantity = :Rbound, nPoints = 33)
```

Comment:
I know I said to comment this out. Let's just delete the test.

Reply:
After defining mix, the test works. Did you still want it out?

Reply:
Oh great. We can keep it then.